### PR TITLE
Removes pm/heap/support from being locked

### PR DESCRIPTION
**Pull request by himanshu0503**
https://github.com/Shippable/beta/issues/534

### CODE REVIEW

**Comment by himanshu0503** [comment]
Not sure how to run this locally, however i've tested the skip logic locally.

File: lockUnlockTeams.sh [hunk 1]
```diff
@@ -55,22 +55,25 @@ change_permissions() {
...
     for repo_name in $TEAM_REPOS; do
-      url="$GITHUB_API_URL/teams/$TEAM_ID/repos/$ORG_NAME/$repo_name"
-      #check if this repo is managed by the team
-      ret=$(curl -s -o /dev/null -w "%{http_code}" -X GET -H "Accept: application/json" -H "Authorization: token $GITHUB_TOKEN" $url)
-      if [ "$ret" == 204 ]; then
-        local ret=$(curl -s -o /dev/null -w "%{http_code}" -X PUT -H "Content-Type: application/json" -H "Accept: application/vnd.github.v3.repository+json" -H "Authorization: token $GITHUB_TOKEN" $url -d "$data")
+      if [[ " ${REPOS_TO_BE_SKIPPED[*]} " != *"$repo_name"* ]]; then
+        url="$GITHUB_API_URL/teams/$TEAM_ID/repos/$ORG_NAME/$repo_name"
+        #check if this repo is managed by the team
+        ret=$(curl -s -o /dev/null -w "%{http_code}" -X GET -H "Accept: application/json" -H "Authorization: token $GITHUB_TOKEN" $url)
```
Comment (ric03uec) [inline lockUnlockTeams.sh]: can you break this into separate lines for readability. something like https://github.com/Shippable/base/blob/master/scripts/_manageMasterIntegrations.sh#L139

File: lockUnlockTeams.sh [hunk 2]
```diff
@@ -55,22 +55,25 @@ change_permissions() {
...
     local permission="$1"
     local data="{\"permission\": \"$permission\"}"
+    local REPOS_TO_BE_SKIPPED=(pm heap support)
     for repo_name in $TEAM_REPOS; do
-      url="$GITHUB_API_URL/teams/$TEAM_ID/repos/$ORG_NAME/$repo_name"
-      #check if this repo is managed by the team
-      ret=$(curl -s -o /dev/null -w "%{http_code}" -X GET -H "Accept: application/json" -H "Authorization: token $GITHUB_TOKEN" $url)
-      if [ "$ret" == 204 ]; then
-        local ret=$(curl -s -o /dev/null -w "%{http_code}" -X PUT -H "Content-Type: application/json" -H "Accept: application/vnd.github.v3.repository+json" -H "Authorization: token $GITHUB_TOKEN" $url -d "$data")
+      if [[ " ${REPOS_TO_BE_SKIPPED[*]} " != *"$repo_name"* ]]; then
```
Comment (ric03uec) [inline lockUnlockTeams.sh]: we should add an `else` block for this `if` to just echo that the repo is being skipped

File: lockUnlockTeams.sh [hunk 3]
```diff
@@ -55,22 +55,25 @@ change_permissions() {
...
+        url="$GITHUB_API_URL/teams/$TEAM_ID/repos/$ORG_NAME/$repo_name"
+        #check if this repo is managed by the team
+        ret=$(curl -s -o /dev/null -w "%{http_code}" -X GET -H "Accept: application/json" -H "Authorization: token $GITHUB_TOKEN" $url)
         if [ "$ret" == 204 ]; then
-          echo "Permission updated to $permission for repository $repo_name"
-          echo "----------------------------------------------"
+          local ret=$(curl -s -o /dev/null -w "%{http_code}" -X PUT -H "Content-Type: application/json" -H "Accept: application/vnd.github.v3.repository+json" -H "Authorization: token $GITHUB_TOKEN" $url -d "$data")
```
Comment (ric03uec) [inline lockUnlockTeams.sh]: break this into separate lines

File: lockUnlockTeams.sh (hunk 2 above)
Comment (himanshu0503) [inline lockUnlockTeams.sh]: Added.

**Pull request merged**